api/src/pcapi/core/fraud/models.py

```diff
@@ -36,6 +37,61 @@ class FraudCheckType(enum.Enum):
 IDENTITY_CHECK_TYPES = [FraudCheckType.JOUVE, FraudCheckType.DMS, FraudCheckType.UBBLE, FraudCheckType.EDUCONNECT]
 
 
+class FraudReasonCode(enum.Enum):
+    # Common to all fraud checks
+    DUPLICATE_USER = "duplicate_user"
+    EMAIL_NOT_VALIDATED = "email_not_validated"
+    MISSING_REQUIRED_DATA = "missing_required_data"
+    NAME_INCORRECT = "name_incorrect"  # The user's name contains unaccepted characters
+    NOT_ELIGIBLE = "not_eligible"
+    AGE_NOT_VALID = "age_is_not_valid"
+
+    # Specific to DMS
+    EMPTY_ID_PIECE_NUMBER = "empty_id_piece_number"
+    ERROR_IN_DATA = "error_in_data"  # The user's data has not passed our API validation
+    REFUSED_BY_OPERATOR = "refused_by_operator"
+
+    # Specific to Ubble
+    # Ubble native errors
+    ID_CHECK_BLOCKED_OTHER = (
+        "id_check_bocked_other"  # Default reason code when the user's ID check is blocked for an unhandled reason
+    )
+    ID_CHECK_DATA_MATCH = "id_check_data_match"  # Ubble check did not match the data declared in the app (profile step)
+    ID_CHECK_EXPIRED = "id_check_expired"
+    ID_CHECK_NOT_AUTHENTIC = "id_check_not_authentic"
+    ID_CHECK_NOT_SUPPORTED = "id_check_not_supported"
+    ID_CHECK_UNPROCESSABLE = "id_check_unprocessable"
+    INVALID_ID_PIECE_NUMBER = "invalid_id_piece_number"
+    BLURRY_VIDEO = "blurry_video"
+    NETWORK_CONNECTION_ISSUE = "network_connection_issue"
+    LACK_OF_LUMINOSITY = "lack_of_luminosity"
+    DOCUMENT_DAMAGED = "document_damaged"
```

Comment:
<!-- Suggestions are specific requests to improve the subject under review. It is assumed that we all want to do what's best, so these comments are never dismissed as “mere suggestions”, but are taken seriously.  -->
**🎯 suggestion:** ‏Trier par ordre alphabétique :)